File: src/main/resources/static/app/model/Product.js
Ext.define("AK.model.Product", {
    extend: 'Ext.data.Model',
    fields: ['productName', 'id', 'store', 'brand', 'model', 'quantity', 'price', 'pricePerProduct'],
    proxy: {
        url: '/products',
        type: 'rest',
        writer: {
            writeRecordId: false,
            writeAllFields: true
        }
    }

})

File: src/main/java/ge/softlab/warehousenew/model/Product.java
package ge.softlab.warehousenew.model;

import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "products")
public class Product {
    @Id
    @GeneratedValue
    private Long id;
    @Column
    private String provider;
    @Column
    private String store;
    @Column
    private String brand;
    @Column
    private String model;
    @Column
    private int quantity;
    @Column
    private int price;
    @Column
    private int pricePerProduct;

}